src/components/CustomerQuota/CheckoutUnsuccessful/CheckoutUnsuccessfulCard.tsx

```diff
@@ -0,0 +1,179 @@
+import {
+  groupTransactionsByCategory,
+  sortTransactions,
+  getLatestTransactionTime,
+} from "../NoQuota/NoQuotaCard";
+import { CampaignConfigContext } from "../../../context/campaignConfig";
+import { BIG_NUMBER } from "../utils";
+
+const MAX_TRANSACTIONS_TO_DISPLAY = 1;
+
+const styles = StyleSheet.create({
+  checkoutItemsList: {
+    marginTop: size(2),
+  },
+});
+
+interface CheckoutUnsuccessfulCard {
+  ids: string[];
+  cart: Cart;
+  onCancel: () => void;
+  quotaResponse: Quota | undefined;
+}
+
+/**
+ * Shows when the user is not able to return tt-token due to:
+ * - Redeemed token and returned token does not have the same id
+ * - Have not redeemed the token but attempted to return
+ */
+export const CheckoutUnsuccessfulCard: FunctionComponent<CheckoutUnsuccessfulCard> = ({
+  ids,
+  cart,
+  onCancel,
+  quotaResponse,
+}) => {
+  const [isShowFullList, setIsShowFullList] = useState<boolean>(false);
+  const { policies: allProducts } = useContext(CampaignConfigContext);
+  const { sessionToken, endpoint } = useContext(AuthContext);
+  const { pastTransactionsResult, loading, error } = usePastTransaction(
+    ids,
+    sessionToken,
+    endpoint
+  );
+  // Assumes results are already sorted (valid assumption for results from /transactions/history)
+  const sortedTransactions = pastTransactionsResult;
+
+  const { showErrorAlert } = useContext(AlertModalContext);
+  useEffect(() => {
+    if (error) {
+      showErrorAlert(error);
+    }
+  }, [error, showErrorAlert]);
+
+  const latestTransactionTime: Date | undefined =
+    (quotaResponse && getLatestTransactionTime(cart)) ?? undefined;
+
+  const translationProps = useTranslate();
+  const { c13nt, i18nt } = translationProps;
+  const transactionsByCategoryMap = groupTransactionsByCategory(
+    sortedTransactions,
+    allProducts || [],
+    latestTransactionTime,
+    translationProps
+  );
+
+  const transactionsByCategoryList = sortTransactions(
+    transactionsByCategoryMap
+  );
+  return (
+    <View>
+      <CustomerCard ids={ids} headerBackgroundColor={color("red", 60)}>
+        <View style={sharedStyles.failureResultWrapper}>
+          <View style={sharedStyles.resultWrapper}>
+            <FontAwesome
+              name="thumbs-down"
+              color={color("red", 60)}
+              style={sharedStyles.icon}
+            />
+            <View style={sharedStyles.statusTitleWrapper}>
+              <AppText
+                style={sharedStyles.statusTitle}
+                accessibilityLabel="checkout-unsuccessful-title"
+                testID="checkout-unsuccessful-title"
+                accessible={true}
+              >
+                {`${c13nt(
+                  "checkoutUnsuccessfulTitle",
+                  undefined,
+                  i18nt("checkoutUnsuccessfulScreen", "unsuccessful")
+                )}`}
+              </AppText>
+            </View>
+            <View>
+              <AppText style={{ marginBottom: size(1) }}>
```

Comment:
What's with this random style here? Does it apply to the CheckoutSuccessCard too?